lib/active_model/serializer/adapter/cached_serializer.rb

```diff
@@ -24,7 +24,7 @@ def cached?
         end
 
         def fragment_cached?
-          @klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except
+          @klass._cache && (@klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except)
```

Comment:
Right maybe except I don't think you can fragment cache if caching isn't available so that seems moot. I think the answer to that question shouldn't be relevant if caching is disabled.